src/contour/TerminalSessionManager.h

```diff
@@ -50,32 +52,38 @@ class TerminalSessionManager: public QAbstractListModel
     [[nodiscard]] int count() const noexcept { return static_cast<int>(_sessions.size()); }
 
     void updateColorPreference(vtbackend::ColorPreference const& preference);
-    display::TerminalDisplay* display = nullptr;
-    TerminalSession* getSession() { return _sessions[0]; }
+
+    display::TerminalDisplay* activeDisplay = nullptr;
```

Comment:
let's not mix fields with functions. Also, I think this should be private, _and_ it is missing a private datamember prefix then (`_activeDisplay`) :)